dialogy/plugins/text/list_entity_plugin/__init__.py

```diff
@@ -170,6 +198,42 @@ def _search(self, transcripts: List[str]) -> List[MatchType]:
             )
         token_list = [search_fn(transcript) for transcript in transcripts]
         return token_list
+    
+
+    # new method based on experiments done during development of channel parser
+    def get_fuzzy_dp_search(self, transcripts: List[str], lang: str) -> List[BaseEntity]:
+        """
+
+
+        """
+        valid_langs = ['hi', 'en']
+        if lang not in valid_langs:
+            raise ValueError(
+                f'Provided language {lang} is not supported by this method at present'
+            )
+        match_dict = {}
+        pos_tags = ["PROPN", "NOUN"]
+        query = '\n'.join(transcripts)
+
+        sentences = self.nlp[lang](query).sentences
+        for sentence in sentences:
+            value = ""
+            for word in sentence.words:
+                if word.upos in pos_tags:
+                    value = value + str(word.text) + " "
+                if value == "":
+                    continue
+            for entity in self.entities[lang]:
+                val = fuzz.ratio(entity, value)/100
+                if val > 0.1 :
+                    match_value = self.entity_dict[lang][entity]
+                    if match_value in match_dict.keys():
+                        if val > match_dict[match_value]:
+                            match_dict[match_value] = val
+                    else:
+                        match_dict[match_value] = val
+
+        match_dict = dict(sorted(match_dict.items(), key=lambda item: item[1], reverse=True)) 
```

Comment:
trailing whitespace. Always use `make lint`.

Reply:
didn't get your point here

Reply:
Instead of:

```python
match_dict = dict(sorted(match_dict.items(), key=lambda item: item[1], reverse=True))
```

```python
SCORE = 1
match_dict = dict(sorted(match_dict.items(), key=lambda item: item[SCORE], reverse=True))
```
makes it more readable.